lib/plugin/gitlab/GitLab.js

```diff
@@ -260,7 +263,12 @@ class GitLab extends Release {
     const body = new FormData();
     const rawData = await fs.promises.readFile(filePath);
     body.set('file', new Blob([rawData]), name);
-    const options = { body };
+    const options = {
+      body,
+      // When using fetch() with FormData bodies, we should not set the Content-Type header.
+      // See: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest_API/Using_FormData_Objects#sending_files_using_a_formdata_object
+      setContentType: false
```

Comment:
Maybe we can check for something like `body instanceof FormData` in the `this.request` method without having to pass this option?

Reply:
We certainly could. This would tend to increase the "magic" in the request function and my taste is to be explicit rather than implicit but it's also not my project. Would you prefer to take that approach? If so, let me know and I'll re-work.

Reply:
No problem - I'll re-work.

Reply:
That's now been done. The diff is indeed now a lot neater.

Reply:
I take your point and am always happy to adjust style to match the project.